<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_cartoon</class>
 <widget class="QDialog" name="dlg_cartoon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cartoon Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>Basic Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Ribbon Shape</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="m_pShapeFlat">
            <property name="text">
             <string>Flat</string>
            </property>
            <attribute name="buttonGroup">
             <string>m_pRibbonShape</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_pShapeRectangular">
            <property name="text">
             <string>Rectangular</string>
            </property>
            <attribute name="buttonGroup">
             <string>m_pRibbonShape</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_pShapeRoundedRectangular">
            <property name="text">
             <string>Rounded Rectangular</string>
            </property>
            <attribute name="buttonGroup">
             <string>m_pRibbonShape</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_pShapeElliptical">
            <property name="text">
             <string>Elliptical</string>
            </property>
            <attribute name="buttonGroup">
             <string>m_pRibbonShape</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>General Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="m_pCheckBoxPerPixelLighting">
            <property name="toolTip">
             <string>High Quality Lighting will have a small impact on rendering speed. It can only be used in conjunction with Vertex/Pixel Shader.</string>
            </property>
            <property name="text">
             <string>High Quality Lighting</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pCheckBoxHelixCylinder">
            <property name="toolTip">
             <string>This draws helices as cylinders.</string>
            </property>
            <property name="text">
             <string>Show Helices as Cylinders</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel1">
              <property name="toolTip">
               <string>If a sheet consists of less residues, it will be rendered as a turn.</string>
              </property>
              <property name="text">
               <string>Min Number of Residues per Sheet</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="m_pSpinBoxSheetRes">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>999999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="textLabel1_2">
              <property name="toolTip">
               <string>If a helix consists of less residues, it will be rendered as turn.</string>
              </property>
              <property name="text">
               <string>Min Number of Residues per Helix</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="m_pSpinBoxHelixRes">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="spacer6_2_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>General Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="m_pCheckBoxHelixArrowHeads">
            <property name="toolTip">
             <string>This draws arrowheads for the helices, too.</string>
            </property>
            <property name="text">
             <string>Draw Helices with Arrowheads</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pCheckBoxShader">
            <property name="toolTip">
             <string>Use of pixel and vertex shaders is very efficient for dynamic scenes.</string>
            </property>
            <property name="text">
             <string>Use Vertex/Pixel Shader for
Fast Protein Dynamics Rendering</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pCheckBoxPeptidePlanes">
            <property name="toolTip">
             <string>This shows simply estimated peptide planes.</string>
            </property>
            <property name="text">
             <string>Show Approximated Peptide Planes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Geometry Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel2_2_2">
            <property name="toolTip">
             <string>Set the diameter scaling factor for the tube parts.</string>
            </property>
            <property name="text">
             <string>Tube Diameter</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="m_pCartoonTubeDiameter">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel2_2_2_2">
            <property name="toolTip">
             <string>Set the width and height scaling factor for the sheets.</string>
            </property>
            <property name="text">
             <string>Sheet Width  /  Height</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="m_pCartoonSheetWidth">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="textLabel3">
            <property name="text">
             <string>/</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>2</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5">
           <widget class="QDoubleSpinBox" name="m_pCartoonSheetHeight">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="textLabel2_2_2_3">
            <property name="toolTip">
             <string>Set the width and height scaling factor for helices, where width is in the direction of the helix axis and the height corresponds to the thickness of the helices.</string>
            </property>
            <property name="text">
             <string>Helix Width / Height</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="m_pCartoonHelixWidth">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="textLabel3_2">
            <property name="text">
             <string>/</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>2</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="5">
           <widget class="QDoubleSpinBox" name="m_pCartoonHelixHeight">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="textlabel">
            <property name="toolTip">
             <string>Set the number of segments between two consecutive atoms. Increases optical quality but adds quickly very many triangles.</string>
            </property>
            <property name="text">
             <string>Segments per Residue</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="m_pSpinBoxSegments">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>4</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="textLabel2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>A high number will make the cartoon look much better but produces a lot more triangles. Even values are recommended.</string>
            </property>
            <property name="text">
             <string>Tube/Cylinder Profile Corners</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="m_pSpinBoxTubeCorners">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2" colspan="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>27</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="6">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2" colspan="4">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>27</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="m_pSpinBoxSheetHelixCorners">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="textLabel2_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>A high number will make the cartoon look much better but produces a lot more triangles. Even values are recommended.</string>
            </property>
            <property name="text">
             <string>Helix/Sheet Profile Corners</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pButtonClose">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>tabWidget2</tabstop>
  <tabstop>m_pCheckBoxHelixCylinder</tabstop>
  <tabstop>m_pCheckBoxHelixArrowHeads</tabstop>
  <tabstop>m_pCheckBoxShader</tabstop>
  <tabstop>m_pCheckBoxPeptidePlanes</tabstop>
  <tabstop>m_pSpinBoxSegments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="m_pRibbonShape"/>
 </buttongroups>
</ui>
